tests/bats/setup_suite.bash

```diff
@@ -47,6 +47,13 @@ validate_prerequisites() {
 
     # Examples: v1, or v1beta1
     export TEST_K8S_RESOURCE_API_VERSION
+
+    # For resource.k8s.io API versions before v1, a ResourceClaim spec looks
+    # slightly differently.
+    export TEST_DEVCLASSNAME_CD_CHANNEL=$'exactly:\n        deviceClassName: compute-domain-default-channel.nvidia.com'
```

Comment:
As this encodes a specific depth of indentation, this is tailored to the template above specifically.

The template is rendered in two different tests.

This isn't nice, but works for now.